plugins/techdocs/src/reader/components/Reader.tsx

```diff
@@ -198,6 +198,7 @@ export const useTechDocsReaderDom = (): Element | null => {
           .md-footer-nav__link { width: 20rem;}
           .md-content { margin-left: 20rem; max-width: calc(100% - 20rem * 2 - 3rem); }
           .md-typeset { font-size: 1rem; }
+          .md-typeset h1, .md-typeset h2, .md-typeset h3 { font-weight: 500; }
```

Comment:
I was looking at the default font-weight for heading elements, and it seems to be `bold`. Looking at material ui typography component (which is not used here but for the rest of backstage) it looks like its a bit different. So 300 for h1, h2 and 400 for h3, h4 etc which seems to be same as what we currently have for TechDocs if I inspect for example the headings of https://demo.backstage.io/docs/default/component/backstage/overview/architecture-overview/. I'm interested in why you went with 500?

Reply:
Yeah, i noticed that.

currently in Techdocs. h1: 1.5625rem, h2: 1.25rem, **h3: 1rem**, **p:1rem** and 300 for h1, h2 and 400 for h3

Material UI has h1: 2rem, h2: 1.5625rem, **h3: 1.25rem**, **p:1rem**  and 300 for h1, h2 and 400 for h3

Because of the font size, you will see there's actually no difference between h3 and p in TechDocs and that 400 weight is not bold but just normal font 

![Screen Shot 2021-11-11 at 3 44 07 am](https://user-images.githubusercontent.com/28945404/141155706-2c052220-ea1d-40d9-955f-59b0029e99bb.png)

refer to https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight
```
font-weight: 400;// normal
font-weight: 700;// bold
```

I set to 500 thinking we want to distinguish but don't want a 'too bold' header.